pkg/cmd/capten.go

```diff
@@ -50,8 +50,11 @@ var showCmd = &cobra.Command{
 func readAndValidClusterFlags(cmd *cobra.Command) (cloudService string, clusterType string, err error) {
 	cloudService, _ = cmd.Flags().GetString("cloud")
```

Comment:
this mapping is wrong

if len(cloudService) == 0 {
		cloudService = "azure"
	} else {
		cloudService = "aws"
	}